less/plugins/adapt-contrib-boxmenu/boxMenuItem.less

```diff
@@ -1,6 +1,6 @@
 .boxmenu-item {
   &__inner {
-    margin: 0 0.5rem 1rem;
+    margin: 0 @menu-item-padding-hoz @menu-item-padding-ver;
```

Comment:
Although variables offer flexibility over hard coded I'm not sure how often these get amended. As the boxmenu doesn't support a proper grid or column layout currently I think these values are pretty set.
I'd also expect padding variables to apply to the inner spacing rather than the gutter.

Reply:
I'm happy for these to remain.